dashboard/test/lib/pd/payment/payment_calculator_csf_test.rb

```diff
@@ -75,5 +75,15 @@ class PaymentCalculatorCSFTest < ActiveSupport::TestCase
       assert_equal 1, summary.num_qualified_teachers
       assert_equal @qualified_teacher, summary.teacher_summaries.find(&:qualified?).teacher
     end
+
+    test 'handle workshops with deleted sections' do
+      @workshop.section.destroy
+      assert_nil @workshop.reload.section
```

Comment:
nit: This assertion isn't (shouldn't) be part of the test (though you may need to reload the workshop before invoking `calculate`).